CLI/test/ressources/outputSignature_3/CLIArguments.txt

```diff
@@ -0,0 +1 @@
+﻿-i input\Callee.rdz.tcbl -o output\ProcedureCall-Public-Signatures.rdz.tcbl -d output\error.txt -e rdz -f PublicSignatures
```

Comment:
Although it works, the naming of files is not representative of their content.
Here are my suggestions :
- 'Callee' could be 'Library'
- 'ProcedureCall-Public-Signatures' could be named 'Client'

Reply:
Or maybe rename 'ProcedureCall-Public-Signatures' to 'Library-public-signatures'

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1631/commits/a858b413562c51d3fc725bd16db27729bd73612f